clippy.toml

```diff
@@ -0,0 +1 @@
+disallowed-types = ["std::collections::HashMap", "std::collections::HashSet"]
```

Comment:
Clippy lets you provide a reason here, something like `{ path = "std::collections::HashMap", reason = "use FxHashMap instead" }`. Might be helpful for our future selves...

Reply:
Updated